packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -1171,24 +1353,34 @@ export class StatsAnalyzer extends EventsScope {
    * @private
    * @param {*} result
    * @param {*} mediaType
+   * @param {*} sendrecvType
    * @returns {void}
    * @memberof StatsAnalyzer
    */
-  private processTrackResult(result: any, mediaType: any) {
-    if (!result || result.type !== 'track') {
+  private processTrackResult(result: any, mediaType: any, sendrecvType: any) {
+    if (!result) {
       return;
     }
-
-    const sendrecvType =
-      result.remoteSource === true ? STATS.RECEIVE_DIRECTION : STATS.SEND_DIRECTION;
-
-    if (sendrecvType === STATS.RECEIVE_DIRECTION) {
-      this.statsResults.resolutions[mediaType][sendrecvType].framesReceived = result.framesReceived;
-      this.statsResults.resolutions[mediaType][sendrecvType].framesDecoded = result.framesDecoded;
-      this.statsResults.resolutions[mediaType][sendrecvType].framesDropped = result.framesDropped;
+    if (result.type !== 'inbound-rtp' && result.type !== 'outbound-rtp') {
+      return;
     }
-
-    if (result.trackIdentifier && !mediaType.includes('audio')) {
+    if (result.frameWidth && result.frameHeight) {
+      this.statsResults.resolutions[mediaType][sendrecvType].width = result.frameWidth;
+      this.statsResults.resolutions[mediaType][sendrecvType].height = result.frameHeight;
+    }
+    if (!mediaType.includes('audio')) {
```

Comment:
This is an optimization to the resolutions object as this was added redundant stats to the resolutions.audio object.